===================================================================
RCS file: /cvsroot/boost/boost/boost/format/feed_args.hpp,v
retrieving revision 1.10
retrieving revision 1.11
diff -u -r1.10 -r1.11
--- boost/boost/boost/format/feed_args.hpp	2003/05/28 11:20:49	1.10
+++ boost/boost/boost/format/feed_args.hpp	2003/08/27 03:24:41	1.11
@@ -34,8 +34,7 @@
 
   template<class Tr, class Ch> inline
   void empty_buf(BOOST_IO_STD basic_ostringstream<Ch,Tr> & os) {
-    static const std::basic_string<Ch, Tr> emptyStr; // avoids 2 cases ( "" and  L"" )
-    os.str(emptyStr);
+    os.str( std::basic_string<Ch, Tr>() );
   }
 
   template<class Ch, class Tr>
